docs/sources/datasources/loki.md

```diff
@@ -66,11 +66,12 @@ You can use the Loki query editor to create log and metric queries.
 
 ### Log browser
 
-With Loki log browser you can easily navigate trough your list of labels and values and construct the query of your choice. Log browser has multi-step selection:
+With Loki log browser you can easily navigate through your list of labels and values and construct the query of your choice. Log browser has multi-step selection:
 
 1. Choose the labels you would like to consider for your search.
 2. Pick the values for selected labels. Log browser supports facetting and therefore it shows you only possible label combinations.
 3. Choose the type of query - logs query or rate metrics query. Additionally, you can also validate selector.
+4. Log volume panel above the log browser shows accumulated log distribution over time for all log queries.
```

Comment:
as far as I understood this section describes the steps required to create a query via the log browser, this line seems out of scope here

Reply:
I will ditch it and update https://grafana.com/docs/grafana/latest/explore/logs-integration/ instead 👍

Reply:
Changed in 1492ff550e660aa6b199d83154efd22c7f1ba2bf